File: golang/search_linear.go
package main

import "fmt"

func search_linear(a []int, t int) int {
    for i, e := range a {
        if e == t {
            return i
        }
    }
    return -1
}

func main() {
    a := []int{3, 4, 7, 5, 1, 6, 8, 2, 9, 0}
    target := 11
    fmt.Println("Array Elements:", a)
    res := search_linear(a, target)

    if res != -1 {
        fmt.Println("Element found at index:", res)
    } else {
        fmt.Println("Element not found")
    }
}

File: python/search_linear.py
a = [2, 0, 4, 5, 1, 3, 6, 7, 8, 9]
key = 8
res = -1

for i in range(0, len(a)):
    if a[i] == key:
        res = i
        break

if res == -1:
    print("Element not found")
else:
    print("Element found at idx:", res)
